### PR TITLE
Update setup.exe

### PR DESCRIPTION
**Pull request by aneatrour**
Added a couple additional refinements for set up process

### CODE REVIEW

**Comment by ostephens** [comment]
Think the line about starting on Windows lives further down the episode in the bit headed "To Run Refine"

File: setup.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ There are versions for Windows, Mac OS X and Linux.
 
 ### Installing and Running OpenRefine
 
-When you download OpenRefine for Windows or Linux from the address above, you are downloading a zip file. To install OpenRefine you simply unzip the downloaded file wherever you want to install the program. This can be to a personal directory or to an applications or software directory - OpenRefine should run wherever you put the unzipped folder. The location has to be a "local" drive as problems have been reported trying to run OpenRefine from a Network drive.
```
Comment (ccronje) [inline setup.md]: Agree with @ostephens keep this as Windows explained further down.

File: setup.md [hunk 2]
```diff
@@ -53,6 +53,6 @@ There are also general and specialist tutorials about using OpenRefine available
 
 ### Downloading the data
 
-You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file. Make a note of the location you save the file.
+You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file that will open in a new browser tab. Be sure to right click or control click in order to save the file. Make a note of the location you save the file.
```
Comment (ccronje) [inline setup.md]: Merge is showing conflict:
<<<<<<< patch-2
You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file that will open in a new browser tab. Be sure to right click or control click in order to save the file. Make a note of the location you save the file.
=======
You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv) by right clicking on the link (NOTE Safari right click and select **download linked file**; Chrome and Firefox right click and select **save link as**). Make a note of the location you save the file to.
>>>>>>> gh-pages

Does this sound ok?
You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file that will open in a new browser tab. Be sure to right click or control click in order to save the file (NOTE Safari right click and select **download linked file**; Chrome and Firefox right click and select **save link as**). Make a note of the location you save the file.

**Comment by ostephens** [comment]
I think the simplest approach here would be to close this PR (sorry @aneatrour ) and submit a new one based on the current setup.md, with the text in the right section

**Comment by ccronje** [comment]
Thanks @aneatrour. Ok, I've created a new PR, will close this one.

**Pull request closed without merging**